R/SeqComparisonTable.R

```diff
@@ -0,0 +1,157 @@
+# SeqComparisonTable.R
+#
+# Purpose: Exploratory System Analysis tool project for BCB420 2019
+# Version: 1.0.0
+# Version history: 1.0.0
+# Date: March 24 2019
+# Author: Heewon Lee (heewon.lee@mail.utoronto.ca)
+# License: MIT
+#
+# Input: A HGNC symbol that the user would like to retrieve
+# the corresponding protein sequence alignments with it's interacting
+# genes according to the STRING database
+# Output: A data frame including the hgnc gene of interest,
+#'     the gene it is being compared to, the protein sequence of
+#'     each resulting protein generated by the genes and
+#'     the consensus sequence and conservation scores based
+#'     off a ClustalW multiple Sequence Alignment.
+# Dependencies: BiocManager, Biostrings, msa
+
+# ====  FUNCTION  =============================================================
+# SeqComparisonTableESA.R
+#'
+#' \code{SeqComparisonTable} Return a data frame containing the alignment data
+#'    between the provided hgnc gene and it's corresponding genes based
+#'    on the STRING database..
+#'
+#' @param hgnc The hgnc symbol of the gene of interest in the form of a
+#'     string. This hgnc gene will have multiple sequence
+#'     alignments being performed on it with all of it's interacting
+#'     proteins based off the STRING database.
+#' @return A data frame including the hgnc gene of interest,
+#'     the gene it is being compared to, the protein sequence of
+#'     each resulting protein generated by the genes and
+#'     the consensus sequence and conservation scores based
+#'     off a ClustalW multiple Sequence Alignment.
+#' @author {Heewon Lee} (aut)
+#' @examples
+#' # Retrieve system database using functions provided from the master package
```

Comment:
Examples with expensive computations or examples that need internet access should be placed in \{dontrun ... } blocks.
(This and other important notes are described in the link above)